src/org/opendatakit/briefcase/model/ProxyConnection.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2011 University of Washington.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.opendatakit.briefcase.model;
+
+/**
+ * This class models a proxy connection.
+ * 
+ * @author rclakmal@gmail.com
+ *
+ */
+public class ProxyConnection {
+	private String host;
+	private Integer port;
+	private String proxyType;
```

Comment:
I agree, the enum is explicit and adds clarity!  Is there any reason that an enum value can't be passed to the constructor for ProxyConnection, e.g. `public ProxyConnection(String host, Integer port, ProxyType proxyType) {` instead of `public ProxyConnection(String host, Integer port, String proxyType) {`?  This way it enforces that the caller has to specify a valid value of proxyType, rather than accepting any String. Might be overly defensive programming, but I think it might be applicable here.